.github/workflows/backend-ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Continuous Integration Backend
+
+on:
+  push:
+    branches: ["main", "master", "develop"]
+  pull_request:
+    branches: ["main", "master", "develop"]
```

Comment:
We don't have a master branch.
If you consider using master, please adjust accordindly.